deps/icu-small/README-SMALL-ICU.txt

```diff
@@ -1,8 +1,8 @@
 Small ICU sources - auto generated by shrink-icu-src.py
 
 This directory contains the ICU subset used by --with-intl=small-icu (the default)
-It is a strict subset of ICU 65 source files with the following exception(s):
-* deps/icu-small/source/data/in/icudt65l.dat : Reduced-size data file
+It is a strict subset of ICU 67 source files with the following exception(s):
+* deps/icu-small/source/data/in/icudt67l.dat : Reduced-size data file
```

Comment:
mention the patch here?

Reply:
that file is auto generated in the update process, but it would be the place to document the patched file.